sw/device/tests/alert_handler_lpg_sleep_mode_alerts.c

```diff
@@ -294,8 +294,8 @@ static void enter_low_power(bool deep_sleep) {
 static void check_wakeup_reason(void) {
   dif_pwrmgr_wakeup_reason_t wakeup_reason;
   CHECK_DIF_OK(dif_pwrmgr_wakeup_reason_get(&pwrmgr, &wakeup_reason));
-  CHECK(pwrmgr_testutils_is_wakeup_reason(
-            &pwrmgr, kDifPwrmgrWakeupRequestSourceFive) == true,
+  CHECK(UNWRAP(pwrmgr_testutils_is_wakeup_reason(
+            &pwrmgr, kDifPwrmgrWakeupRequestSourceFive)),
```

Comment:
Should we have an explicit test for true: `CHECK(UNWRAP(foo) == true)`?

Reply:
I think you are right, because the unwrapped value is not explicitly a boolean.

Reply:
Done